src/uucore/src/lib/features/fsext.rs

```diff
@@ -656,28 +656,29 @@ impl FsMeta for StatFs {
             not(target_arch = "s390x"),
             target_pointer_width = "64"
         ))]
-        return self.f_bsize;
+        return self.f_bsize; // i64 -> i64 - OK
```

Comment:
I'm not sure those comments are needed, the Rust compiler is here to tell us if this is ok or not.

Reply:
I put it there to help the reviewer/developer that reads this code to understand what happens. There are a lot of #[cfg(..)] around there and its not directly visible what the main point in the three cases is. But I'll remove it if you insist on.

Reply:
I was just expressing a preference, I'll let the coreutils maintainers weight on this.